docs/src/main/sphinx/release/release-375.md

```diff
@@ -0,0 +1,68 @@
+# Release 375 (24 Mar 2022)
+
+## General
+
+* Change behavior of `ALTER TABLE qualified_name TO unqualified_name` to leave
+  the table in the schema. This is backwards-incompatible behavioral change.
+  ({issue}`11282`)
+* Add processed input bytes and rows to query events in event listener. ({issue}`11623`)
```

Comment:
@ebyhr added here since I dont think we maintain a separate section for event listener..

Reply:
We've used "Event listener", "General" or "SPI" section for event listener changes in the past release notes. Moving to general looks fine.

Reply:
Should be SPI IMO.

Reply:
ok .. will move